.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+# Docs: https://help.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  # Enable version updates for python
+  - package-ecosystem: "pip"
+    directory: "/requirements"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 16
+    rebase-strategy: "disabled"
```

Comment:
we can get rid of this, this is used to reduce the number of builds but there aren't that many builds on sync-engine